packages/Calendar/stories/cypress.stories.js

```diff
@@ -2,13 +2,16 @@ import React from "react";
 import { storiesOf } from "@storybook/react";
 import moment from "moment";
 import L10n from "@paprika/l10n";
+import { getStoryName } from "storybook/storyTree";
 import SingleDateCalendar from "../src/SingleDateCalendar";
 import DateRangeCalendar from "../src/DateRangeCalendar";
 import { START_DATE } from "../src/tokens";
 
+const storyName = getStoryName("Calendar");
+
 const noop = () => {};
 
-storiesOf("Calendar / cypress", module)
+storiesOf(`${storyName}/Backyard/Tests/Cypress`, module)
```

Comment:
nit this change necessary? Harder to read and track which story you are writing for now when developing....

Reply:
Too late, they've all been changed:
https://github.com/acl-services/paprika/pull/546

The purpose is to control the order and categories in one single place in the code (`./.storybook/storyTree.js`).